src/containers/Application/ApplicationWrapper.tsx

```diff
@@ -41,9 +43,11 @@ const ApplicationWrapper: React.FC = () => {
       <Route exact path={`${path}/summary`}>
         <ApplicationSummary structure={structure} />
       </Route>
-      <Route exact path={`${path}/summary/submission`}>
-        <ApplicationSubmissionNEW structure={structure} />
+      <Route exact path={`${path}/submission`}>
+        <ApplicationSubmission structure={structure} />
       </Route>
+      <Route exact path={`${path}/reviewTest`}>
+        <ReviewPageWrapperTest structure={structure} />
```

Comment:
Please add TODO: Remove?

Reply:
this is some messed up committing in the new pr it' sonly in test